docs/book/src/tasks/experimental-features/cluster-classes.md

```diff
@@ -0,0 +1,497 @@
+# Cluster Class
+
+The ClusterClass feature introduces a new way to create clusters which reduces boilerplate required and allows flexible and powerful customization of CAPI-created clusters.
+ClusterClass is implemented on top of pre-existing CAPI interfaces and will 
+
+This tutorial covers using Cluster API and Cluster Class to create a Kubernetes cluster and how to perform a one-touch Kubernetes update procedure using ClusterClass features.
+
+<aside class="note warning">
+
+<h1>Note</h1>
+
+This guide covers creating a ClusterClass cluster with the CAPD provider.
+</aside>
+
+## Installation
+
+### Common Prerequisites
+This guide is based on having the same pre-requisites as the CAPI [quick start guide].
+That is:
+- Install and setup [kubectl]  and [clusterctl]
+- Install [Kind] and [Docker]
+
+ClusterClass is currently behind a feature gate that needs to be enabled. To do so run:
```

Comment:
Let's explicity say that in this tutorial we also use another experimental feature, CRS, so we need to enable both
(I saw it is down in the paragraph, but we need something before the export)